plugins/katello/3.3/release_notes/release_notes.md

```diff
@@ -4,16 +4,26 @@ title: Release Notes
 version: 3.3
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.3 (Baltic Porter) Release Notes
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
+
+### Publish "Latest" Version in Composite Content Views
+
+### Capsule Download Policy
+
+### Tracer Support
+
```

Comment:
After updating some packages on a host sometimes a service will need to be restarted in order for the new version to take affect.  Tracer is a utility to help detect these services and Katello 3.3 supports hosts reporting this information.  Simply update to the newest katello-agent and these reports will be sent whenever a package is updated.  You can see what services need to be restarted and restart them via Remote Execution from the new Content Host Details Traces page.